floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -144,7 +147,8 @@ class QueryMethodWriter implements Writer {
   ) {
     final queryableName = _queryMethod.queryable.name;
     final isView = _queryMethod.queryable is View;
-    final parameters = StringBuffer()..write("'${_queryMethod.query}', ");
+    final parameters = StringBuffer()
+      ..write('r""" ${_queryMethod.query} """, ');
```

Comment:
❓ Did you add the leading and trailing space on purpose?
```suggestion
      ..write('r"""${_queryMethod.query}""", ')
```

Reply:
See above